lib/python/pyflyby/_autoimp.py

```diff
@@ -1425,6 +1425,7 @@ def _try_import(imp, namespace):
             logger.info("  => Failed: pre-existing %r (%r) differs from imported %r",
                         name0, preexisting, name0)
             return False
+    send_comm_message("pyflyby.missing_imports", {"missing_imports": stmt})
```

Comment:
Don't do this when not in ipython notebook.  Don't do this in ipython CLI or when not in ipython.

Let's try to move this to somewhere else so it's not part of a core function.

Reply:
Resolved along with earlier review comment ("don't do anything when using ipython CLI")